Source/WebCore/platform/ScrollbarsController.h

```diff
@@ -92,7 +92,8 @@ class ScrollbarsController {
 
     WEBCORE_EXPORT virtual String horizontalScrollbarStateForTesting() const { return emptyString(); }
     WEBCORE_EXPORT virtual String verticalScrollbarStateForTesting() const { return emptyString(); }
-    
+    WEBCORE_EXPORT virtual String scrollbarsControllerTypeForTesting() const { return emptyString(); }
```

Comment:
If you add type trait support to all the subclasses, you don't need this; you could have `Internals::scrollbarsControllerTypeForNode` do the `is<>` checks to return the right type string.

Reply:
Fixed.